firmware/boron-ins-fsm/src/debugFlags.h

```diff
@@ -0,0 +1,6 @@
+#ifndef DEBUG_FLAGS_H
```

Comment:
Can you please add a line in the CHANGELOG?
And in the description or a comment in this PR, describe the testing that you've done to feel confident that this works.